src/cldfbench/commands/media.py

```diff
@@ -0,0 +1,376 @@
+'''
+Download {and pack} (filtered) media files {and prepare release on Zenodo}
+
+General workflow:
+ (1) call cldfbench media -l
+       to get an overview of all media files incl. mimetypes and sizes
+ (2) call cldfbench media
+       to download all media files
+     or
+     call cldfbench media --mimetype foo,bar
+       to download only those media files which matches mimetypes
+       - possible filter(s) could be:
+         -- file extentions like wav,mp3,pdf, ...
+         -- mimetypes like audio/ogg,audio/x-wav,application/pdf, ...
+         -- mimetype classes like audio/,video/, ...
+ (3) call cldfbench media --create-release --parent-doi {--mimetype foo,bar}
+       {to download and} create the release directory with the zipped media files,
+       a README.md and zenodo.json
+       - step (2) and (3) can be combined
+       - a --parent-doi is required
+ (4) check README.md and zenodo.json and modify if necessary
+ (5) upload the files media.zip and README.md to Zenodo and remember the deposit ID (number after last slash)
+       - it is necessary to log in via correct zenodo user and to have the corresponding access token
+         in your environment
+       - it is only necessary to fill in required fields with provisional data - see step (6)
+ (6) call cldfbench media --upload-zenodo deposit_ID
+       to update the metadata of the previous uploaded reelease
+'''
+
+import collections
+import html
+import mimetypes
+import os
+import pathlib
+import subprocess
+import threading
+import time
+import tqdm
+import zenodoclient
+import zipfile
+
+from urllib.request import urlretrieve
+
+from cldfbench.cli_util import add_dataset_spec, get_dataset
+from cldfbench.datadir import DataDir
+from cldfbench.metadata import get_creators_and_contributors
+from clldutils import jsonlib
+from clldutils.clilib import PathType
+from clldutils.misc import format_size, nfilter
+from clldutils.path import md5, git_describe
+from csvw.dsv import UnicodeWriter
+from zenodoclient.api import Zenodo, API_URL, API_URL_SANDBOX, ACCESS_TOKEN
+from zenodoclient.models import PUBLISHED
+
+
+MEDIA = 'media'
+ZENODO_FILE_NAME = 'zenodo.json'
```

Comment:
Why not `.zenodo.json`?

Reply:
per default it would be an hidden file and acc. to workflow one should check it before usage, thus it's easier

Reply:
Ok